### DISCUSSION TITLE
It's run with only local server or also local network?

### DISCUSSION THREAD

**Discussion by SnehalLodaliyaIT** [mockoon/mockoon#388] [Help] [answered]
Hello everyone,

I started to use Mockoon and I think it is a great tool. Thank you for this. ;)
I think i have to read it's documentation in detail i tried for that but i still can't  understand exactly for running environment.
If i want to use with a team for single project and i think mockoon supports for only local network.
There are any another solution for mockoon use with team?

**Comment by 255kb** [comment, marked as answer]
Hello @SnehalLodaliyaIT Thank you for using Mockoon 😃 
While Mockoon doesn't have an option to deploy mocks online, we have teams successfully using Mockoon on their local network. 
All mocks that Mockoon are running are available on all network adapters. If your computer is reachable on the local area network, your coworkers could call an API mock by entering your computer's local IP: `192.168.x.x:port/path`. 
You could also use the [CLI](https://github.com/mockoon/cli) to make the running mock available on a server that everybody can reach.

**Reply by michaelsogos** [reply]
Hello there,

We have same needs.
Is possible to install mockoon in on-premise server and use the windows\mac electron app to configure it remotely?
To work on a team is not enought just call the endpoint with any client (in our case mobile app own developed), but also configure endpoints as we need.
From our understanding seems that electron app can just run mock server locally but cannot "configure" new mock\stubs on remote installation; isn't it?

**Reply by 255kb** [reply]
Hello, you are right, Mockoon app cannot be remotely configured or be used to configure a running CLI remotely.

**Reply by michaelsogos** [reply]
Do you have plan for it?

**Reply by 255kb** [reply]
Not yet. It hasn't been requested a lot (you are among the first). Also, considering other requests already on the roadmap (GraphQL, wesocket, etc), it wouldn't be a top priority. But as always, if we note interest from more people we may reconsider and plan it.

**Reply by kliushnichenko** [reply]
Such a great tool, fancy and user-friendly UI. Sadly, it is not possible to run as a standalone server with UI. +1 for this feature

**Reply by guawali** [reply]
Hello 255kb, And our team really needs it , want it to be configured remotely.
I hope you can consider it , thanks

**Reply by avrezvanov** [reply]
We also need a command tool +1.

**Reply by m-roberts** [reply]
I have created an Issue for such a workflow: https://github.com/mockoon/mockoon/issues/901

**Reply by guawali** [reply]
Hello 255kb, I am a Chinese user, Such a great tool,  For this tool 666,  Hopefully it can run through the server.  +1 for this feature，thanks
